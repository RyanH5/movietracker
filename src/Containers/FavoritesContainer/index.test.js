import {FavoritesContainer} from './index';
import { shallow } from 'enzyme';
import React from 'react';

describe('FavoritesContainer', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<FavoritesContainer />);

    expect(wrapper).toMatchSnapshot();
  });
});