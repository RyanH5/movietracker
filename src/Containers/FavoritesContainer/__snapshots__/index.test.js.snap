// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesContainer should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FavoritesContainer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        <h3>
                Sorry, no faves.
        </h3>
        <NavLink
                activeClassName="active"
                aria-current="page"
                className="nav logout"
                to="/"
        >
                Back to Movies
        </NavLink>
</div>,
      "className": "favorites-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Sorry, no faves.
</h3>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav logout"
            to="/"
>
            Back to Movies
</NavLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sorry, no faves.",
          },
          "ref": null,
          "rendered": "Sorry, no faves.",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Back to Movies",
            "className": "nav logout",
            "to": "/",
          },
          "ref": null,
          "rendered": "Back to Movies",
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          <h3>
                    Sorry, no faves.
          </h3>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="nav logout"
                    to="/"
          >
                    Back to Movies
          </NavLink>
</div>,
        "className": "favorites-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Sorry, no faves.
</h3>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="nav logout"
              to="/"
>
              Back to Movies
</NavLink>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sorry, no faves.",
            },
            "ref": null,
            "rendered": "Sorry, no faves.",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "Back to Movies",
              "className": "nav logout",
              "to": "/",
            },
            "ref": null,
            "rendered": "Back to Movies",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
